plugins/modules/aci_interface_policy_spine_policy_group.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Anvitha Jain <anvjain@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_interface_policy_spine_policy_group
+short_description: Manage fabric interface policy spine policy groups (infra:SpAccPortGrp)
+description:
+- Manage fabric interface policy spine policy groups on Cisco ACI fabrics.
+options:
+  policy_group:
+    description:
+    - Name of the spine policy group to be added/deleted.
+    type: str
+    aliases: [ name, policy_group_name, spine_policy_group_name ]
+  description:
+    description:
+    - Description for the spine policy group to be created.
+    type: str
+    aliases: [ descr ]
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+  link_level_policy:
+    description:
+    - Choice of link_level_policy to be used as part of the spine policy group to be created.
+    type: str
+    aliases: [ link_level_policy_name ]
+  link_flap_policy:
+    description:
+    - Choice of link_flap_policy to be used as part of the spine policy group to be created.
+    type: str
+    aliases: [ link_flap_policy_name ]
+  cdp_policy:
+    description:
+    - Choice of cdp_policy to be used as part of the spine policy group to be created.
+    type: str
+    aliases: [ cdp_policy_name ]
+  mac_sec_policy:
+    description:
+    - Choice of mac_sec_policy to be used as part of the spine policy group to be created.
+    type: str
+    aliases: [ mac_sec_policy_name ]
+  attached_entity_profile:
+    description:
+    - Choice of attached_entity_profile to be used as part of the spine policy group to be created.
+    type: str
+    aliases: [ aep_name, aep ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+notes:
+seealso:
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC classes B(infra:SpAccPortGrp).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Anvitha Jain (@anvjain)
+"""
+
+EXAMPLES = r"""
+- name: Create a Spine Interface Policy Group
```

Comment:
It would be good to add an actual name/value in the examples below for the attributes below?

Reply:
changed